lib/Server.js

```diff
@@ -980,6 +985,9 @@ class Server {
   }
 
   serveMagicHtml(req, res, next) {
+    const getFilenameFromUrl =
+      require('webpack-dev-middleware/dist/utils/getFilenameFromUrl').default;
```

Comment:
I think we should add this method to public `webpack-dev-middleware` API here https://github.com/webpack/webpack-dev-middleware/blob/master/src/index.js#L78, in future we should avoid import non public API

Reply:
Makes sense, will do 👍🏻